cabal.project

```diff
@@ -31,6 +31,7 @@ packages: ./cardano-ping
           ./ouroboros-network-protocols
           ./ouroboros-network-testing
           ./ouroboros-consensus
+          ./ouroboros-consensus-tutorials
```

Comment:
The tutorials were already included in `ouroboros-consensus:tutorials`, which now fails to build -- we should probably remove the sublib from `ouroboros-consensus`

Reply:
Ups, indeed.